ElementaryTasks/Loops.cs

```diff
@@ -0,0 +1,134 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace ElementaryTasks
+{
+    class Loops
+    {
+        public void SumOfEvenValues()
+        {
+            int sum = 0;
+            int count = 0;
+            for (int i = 1; i < 99; i++)
+            {
+                if (i % 2 != 0)
```

Comment:
Please revert statement to remove 'continue'

Reply:
done